src/components/Profile/components/SchedulePanel/index.tsx

```diff
@@ -48,11 +52,22 @@ const GordonSchedulePanel = ({ profile, myProf }: Props) => {
     });
   }, [profile.AD_Username]);
 
+  const toggleIsScheduleOpen = () => {
+    setIsScheduleOpen((wasOpen) => {
+      localStorage.setItem('scheduleOpenKey', String(!isScheduleOpen));
+      return !wasOpen;
+    });
+  };
+
   return loading ? (
     <GordonLoader />
   ) : (
     <>
-      <Accordion TransitionProps={{ unmountOnExit: true }}>
+      <Accordion
+        expanded={!isScheduleOpen}
```

Comment:
I'm confused by this. `isScheduleOpen` default to `true` when the user hasn't set anything yet, so by default this will set the `expanded` prop to false. Shouldn't it be the reverse?

Reply:
For some reason when i had scheduleOpenKey in quotes, setting isScheduleOpen to false made it expanded by default.  After fixing that and removing the quotes I had to change it back. Could have also been faulty testing on my part.